lib/bloomfilter.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable */
+// TODO: Remove previous line and work through linting issues at next edit
+
+'use strict';
+
+var BloomFilter = require('bloom-filter');
+var BufferReader = require('./encoding/bufferreader');
+var BufferWriter = require('./encoding/bufferwriter');
+
+/**
+ * A constructor for Bloom Filters
+ * @see https://github.com/bitpay/bloom-filter
+ * @param {Buffer} payload
+ */
+BloomFilter.fromBuffer = function fromBuffer(payload) {
```

Comment:
Maybe it's better to use util.inherits instead of patching class directly?

Reply:
I just copy-pasted this file as is. Do you think we need to refactor it too?

Reply:
Well, I have a feeling that we never change it other way

Reply:
What are the minimal requirements for JS here? May I use destructors, classes and so on?

Reply:
It's ES5. We need to bump major version otherwise

Reply:
Decided to keep it as is.